scripts/family_tail_acceptance_570.py

```
@@ -0,0 +1,602 @@
+
+        # ── P-5 three LLM/text faces: real rendered products, pos+neg each ──
+        # 1) 召对认账 brief（真实 vocabulary 渲染产物；空 brief 不计过）
+        ref_name = cabinet_name
+        # Prefer a name that still has referenceable force-promulgated bare dossier.
+        candidates = db.list_referenceable_dossiers(bare_name, state.turn)
+        if not candidates:
+            candidates = db.list_referenceable_dossiers(ref_name, state.turn)
+        brief = render_referenceable_dossier_brief(candidates)
+        p4_brief = _p4_scan("召对认账brief", brief)
+
+        # 2) 密奏：真实 record_dossier_progress → list_dossier_progress 产物
```

Ruff (0.16.1)

[warning] Comment contains ambiguous `：` (FULLWIDTH COLON). Did you mean `:` (COLON)?

(RUF003)